packages/fxa-event-broker/lib/result.ts

```diff
@@ -0,0 +1,12 @@
+/* Pattern captured from blog post: https://spin.atomicobject.com/2018/02/23/error-either-typescript/
+ */
+
+export type Result<T> = T | Error;
```

Comment:
Nice, if I squint my eyes a bit I can pretend we're using Rust.

Reply:
If you really want to pretend its Rust, there's even https://github.com/bcherny/tsoption ;)